markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 15 Line 11_
```markdown

***

Text

---------------------------------------

Between horizontal lines
```
Between horizontal lines review comment 1

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._

Reply:
response to comment 1